src/main/java/org/dinosaur/foodbowl/domain/comment/application/CommentService.java

```diff
@@ -56,4 +57,18 @@ public void updateComment(Long commentId, Long memberId, CommentUpdateRequest co
 
         comment.update(commentUpdateRequest.getMessage());
     }
+
+    @Transactional
+    public void deleteComment(Long commentId, Long memberId) {
+        Comment comment = commentRepository.findById(commentId)
+                .orElseThrow(() -> new FoodbowlException(COMMENT_NOT_FOUND));
+        Member member = memberRepository.findById(memberId)
+                .orElseThrow(() -> new FoodbowlException(MEMBER_NOT_FOUND));
+
+        if (comment.isNotWrittenBy(member)) {
```

Comment:
```suggestion
        if (!comment.isWrittenBy(member)) {
```
메서드는 긍정문으로 작성하는게 어떨까요? 🤔

Reply:
zoayo ~